packages/core/stories/components/BrandBanner/BrandBanner.stories.mdx

```diff
@@ -0,0 +1,69 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+
+import generateTitle from '../../util/generateTitle';
+import * as stories from './BrandBanner.stories.js';
+
+<Meta
+  title={generateTitle('BrandBanner')}
+  parameters={{
+    viewMode: 'docs'
+  }}
+/>
+
+# Brand Banner
+
+The brand banner identifies official websites of the Commonwealth of Massachusetts. It ensures the visitors of getting information or performing transactions on an official and secure government website. 
+
+<Canvas withSource="none" >
+  <Story story={stories.brandBannerExample} />
+</Canvas>
+
+> It’s required for the mass.gov domain and all its subdomains (mass.gov domain policy), and recommends all other .gov state websites using it as a consistent and trustworthy identifier for the public users. It's prohibited for non Massachusetts state government websites to use this banner.
```

Comment:
Changed from passive voice:
"According to our domain policy, all sites under the mass.gov domain and its subdomains *must* use the brand banner. Other .gov state websites *should* use it as a consistent and trustworthy identifier. Any entity not directly under the authority of the government of Massachusetts is prohibited from using the banner."